common/views/components/ItemLink/ItemLink.tsx

```diff
@@ -1,4 +1,5 @@
 import NextLink, { LinkProps } from 'next/link';
+import { useRouter } from 'next/router';
```

Comment:
needed?

Reply:
Apologies, I was trying something to stop prop drilling, and had some hangover.